src/ReactiveUI.Validation/Collections/ValidationText.cs

```diff
@@ -13,79 +15,124 @@ namespace ReactiveUI.Validation.Collections
     /// </summary>
     public class ValidationText : IEnumerable<string>
     {
-        private readonly List<string> _texts = new List<string>();
+        /// <summary>
+        /// The empty validation text singleton instance, contains a single empty string.
+        /// </summary>
+        public static readonly ValidationText Empty = new ValidationText(Array.Empty<string>());
+
+        // TODO When Add() & Clear() are obsolesced this should be made read-only.
+        private /* readonly */ string[] _texts;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ValidationText"/> class.
         /// </summary>
+        [Obsolete("Calling the constructor is deprecated, please use ValidationText.Create() overload instead.")]
```

Comment:
tip: consider marking the obsolete stuff with `[ExcludeFromCodeCoverage]`, this will prevent codecov from complaining.